linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs/tests/rhel7_correct_value.pass.sh

```diff
@@ -0,0 +1,3 @@
+# platform = Red Hat Enterprise Linux 7
+
```

Comment:
This scenario is specific for CIS profile, therefore we need to add the profile ID into `profiles` metadata.